lib/sendgrid/helpers/eventwebhook/eventwebhook.rb

```diff
@@ -1,4 +1,6 @@
-require 'starkbank-ecdsa'
+require 'digest'
+require 'openssl'
+
```

Comment:
Might want to `require 'base64'` here too. Looks like you're getting it by default because `SendGrid::Attachment` [requires it here](https://github.com/sendgrid/sendgrid-ruby/blob/master/lib/sendgrid/helpers/mail/attachment.rb#L1) but you don't want to be confused if that changes for some reason, or the order of requires breaks things.